### DISCUSSION TITLE
How to set a custom error page for reject request

### DISCUSSION THREAD

**Discussion by xx-zhang** [corazawaf/coraza#1249] [Q&A]
How to set a custom error page for reject request . 

such as this png show, the defalt error_page:403 response is None, but i want to custom a errror_page to set some ResponseBody, how to generate it. my using example is `example/http_server`. 

![image](https://github.com/user-attachments/assets/362fda37-12e2-45a4-ab62-51ef5d6f4f84)

**Comment by jcchavezs** [comment]
This usually isn't attribution of the WAF, are you using any proxy in front of this?

**Reply by xx-zhang** [reply]
> This usually isn't attribution of the WAF, are you using any proxy in front of this?

@jcchavezs no any middleware/proxy in fornt of http_server.
